sequencer/api/state_relay_server.toml

```diff
@@ -0,0 +1,19 @@
+[meta]
+NAME = "hotshot_state_relay_server"
+DESCRIPTION = "A relay server for storing and serving the light client states and their signatures"
+FORMAT_VERSION = "0.1.0"
+
+[route.poststatesignature]
+PATH = ["state/post"]
```

Comment:
Nit: it would be more conventional for the route to just be `state` with a method of `POST`. Putting `post` also in the route path is redundant

Reply:
changed in [1dde57c](https://github.com/EspressoSystems/espresso-sequencer/pull/1016/commits/1dde57c61e4683b1fde67bb5bde9e666a563311b)